apps/condo/domains/resident/constants/constants.js

```diff
@@ -168,6 +168,9 @@ const MAX_RESIDENT_DISCOVER_CONSUMERS_BY_WINDOW_SEC = 10
 const RESIDENT_FIND_ORGANIZATIONS_WINDOW_SEC = 24 * 60 * 60 // seconds
 const MAX_RESIDENT_FIND_ORGANIZATIONS_BY_WINDOW_SEC = 100
 
+const RESIDENT_SUGGEST_SERVICE_PROVIDER_WINDOW_SEC = 24 * 60 * 60 // seconds
```

Comment:
Please be more informative with time comments, so we can avoid extra calculation while reading

```suggestion
const RESIDENT_SUGGEST_SERVICE_PROVIDER_WINDOW_SEC = 24 * 60 * 60 // 1 day in seconds
```